README.md

```diff
@@ -33,8 +37,30 @@ In this guide you may find the following steps:
     * Link to download manual: J-Link manual [Download](https://www.segger.com/downloads/jlink/)
 * Download and install [Tera Term](https://osdn.net/projects/ttssh2/releases/) (recommended)
 * Download this repository either via a direct [Download](https://github.com/Infineon/getstarted-optiga-trust-m/releases) or clone using `git clone --recursive`.
+## Quickstart XMC1400
```

Comment:
I don't see a need to have two almost identical descriptions. Please update the existing one to reflect both options.